### PR TITLE
DrivAerML: Fourier+no-EMA+T_max=30 — lr sweep (3e-4, 8e-4)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The baseline lr=5e-4 was chosen early in training (only 6 epochs, ~35 min). At longer training horizons, the optimal learning rate may differ. Lower lr (3e-4) provides finer gradient steps that may converge to better minima at late training. Higher lr (8e-4) was the best DrivAerML config before Fourier was added (PR #2467: 56.91%) — with Fourier features enabled, it may unlock faster early convergence and better final performance. Both extremes should be tested against the 5e-4 baseline at longer training.

## Instructions

Run **two sequential commands** — lr=3e-4 first, then lr=8e-4. Use `--wandb_group drivaerml-lr-sweep` to group them.

**Run 1 — lr=3e-4 (conservative, finer steps):**
```bash
cd target/icml2026 && python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 3e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group drivaerml-lr-sweep
```

**Run 2 — lr=8e-4 (aggressive, faster convergence):**
```bash
cd target/icml2026 && python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 8e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group drivaerml-lr-sweep
```

Split time roughly 50/50 between the two runs (e.g. ~90 min each if you have a 180-min budget).

**CRITICAL OOM-prevention flags — do NOT omit:**
- `--batch-size 1`
- `--drivaerml-train-surface-points 50000`
- `--drivaerml-eval-surface-points 50000`
- `--max-train-batches 394`
- `--max-eval-batches 200`
- `--epochs 999` (MANDATORY — rely on timeout, not epoch limit)

Report `val_primary/surface_rel_l2_pct`, best epoch, and W&B run ID for each run.

## Baseline

- **Dataset:** DrivAerML
- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best:** 33.65% val / 34.00% test (PR #2543, lr=5e-4, T_max=30, 6 epochs, ~35 min)
- **External target:** <3.71%
- **Baseline W&B run:** xm765o85
- **Baseline reproduce:** `cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200`

**Target:** Beat val_primary/surface_rel_l2_pct < 33.65%

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2561 because Superseded: mitsuha reassigned to #2607 (DrivAerML T_max=20).
